op-batcher/batcher/config.go

```diff
@@ -96,6 +96,19 @@ type CLIConfig struct {
 	// ActiveSequencerCheckDuration is the duration between checks to determine the active sequencer endpoint.
 	ActiveSequencerCheckDuration time.Duration
 
+	// SequencerRpc is the HTTP provider URL for the sequencer / block builder's execution engine.
+	SequencerRpc string
```

Comment:
This is not needed, there's a `OP_BATCHER_L2_ETH_RPC` which configures the rpc for op-geth on sequencer, and you can use the corresponding initialized clients: https://github.com/ethereum-optimism/optimism/blob/develop/op-service/dial/active_l2_provider.go#L22

Reply:
according to the flag definition that one can be a comma separated list of l2 nodes and I assumed not all of them would be the sequencer.

Reply:
talked offline, the activeL2Provider returns the client that points to the leader sequencer

Reply:
ok changed, PTAL!